### DISCUSSION TITLE
support editing props for `editable.root`

### DISCUSSION THREAD

**Discussion by s97712** [chakra-ui/zag#1712] [Ideas]
Hope to provide the editing props to change the editable state through external means, such as contextmenu.

**Comment by anubra266** [comment]
@s97712 Could you give more info on what you're trying to achieve?

**Comment by s97712** [comment]
I am using Ark's editable, which by default is triggered only by focus and double-click. If there were an editing context, I could have better control over how to initiate the edit, such as by pressing F2 or using the context menu.

**Comment by segunadebayo** [comment]
This is a good addition. I've added support for this in Zag.js [here](https://github.com/chakra-ui/zag/commit/f558b8f402946b6c8222e1f7ea352fac1dc4f017)

Will be available in Ark after the upstream release
